source/deployment/opennebula_installation/frontend_installation.rst

```diff
@@ -260,6 +260,9 @@ The following table lists some notable paths that are available in your Front-en
 | ``/var/lib/one/remotes/tm/``        | Transfer Manager Driver scripts                                                      |
 +-------------------------------------+--------------------------------------------------------------------------------------+
 
+
+.. _oneadmin_credentials:
```

Comment:
This should go on the manage_users guide. Also, please review the process, at least there is one step missing. Restarting the scheduler